recipes/blorg

```diff
@@ -0,0 +1 @@
+(blorg :fetcher github :branch "main" :repo "emacs-love/blorg")
```

Comment:
Just updating the recipe here.  I assume `main` is your default branch, in which case it doesn't need to be specified:
```suggestion
(weblorg :fetcher github :repo "emacs-love/weblorg")
```